js/main.js

```diff
@@ -0,0 +1,163 @@
+var title = document.getElementById('title');
+var wishlist = document.getElementById('wishlist');
+var homeButton = document.getElementById('home-button');
+var startButton = document.getElementById('start-btn');
+var main = document.querySelector('main');
+var header = document.querySelector('header');
+var background = document.querySelector('body');
+var splashScreen = document.querySelector('.splash-screen');
+var unListed = document.getElementById('list');
+var searchBar = document.getElementById('search-coin');
+var favoriteOn = false;
+
+function getCryptoCoins() {
+  var xhr = new XMLHttpRequest();
+  xhr.open('GET', 'https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&order=market_cap_desc&per_page=100&page=1&sparkline=false');
+  xhr.responseType = 'json';
+  xhr.addEventListener('load', function () {
+    xhr.response.forEach(item => {
+      var coin = {
+        id: item.id,
+        name: item.name,
+        image: item.image,
+        symbol: item.symbol,
+        currentPrice: item.current_price,
+        percent: item.price_change_percentage_24h,
+        isFavorite: false
+      };
+      coins.push(coin);
+    });
+
+    var favCoinsStr = localStorage.getItem('Crypto-Coins');
+    var favCoins = JSON.parse(favCoinsStr);
+    var favCoinsMap = new Map(favCoins.map(key => [key.symbol, key.isFavorite]));
```

Comment:
You should be doing localStorage saving and accessing in `data.js`. I recommend looking at `javascript-local-storage` for review on how to deal with local storage effectively.